programs/survival/carried_totem.sc

```diff
@@ -0,0 +1,21 @@
+// Allow a Totem of Undying anywhere in the inventory to activate
+
+__config() -> ({ 'stay_loaded' -> true });
```

Comment:
please leave config to specify for others if needed.

Reply:
done